style.css

```diff
@@ -1,4 +1,352 @@
+    width: 45px;
+    height: auto;
+}
+
+.pumpkin {
+    position: absolute;
+    top: 107px;
+    left: 233px;
+}
+
+.pumpkin img {
+    width: 45px;
+    height: auto;
+}
+
+.grimReaper {
+    position: absolute;
+    top: 229px;
+    right: 200px;
+}
+
+.grimReaper img {
+    width: 45px;
+    height: auto;
+}
+
+.rock {
+    position: absolute;
+    top: 228px;
+    right: 173px;
+    z-index: 10;
+}
+
+.rock img {
+    width: 92px;
+    height: auto;
+}
+
+.cross {
+    position: absolute;
+    top: 27px;
+    right: 218px;
+}
+
+.cross img {
+    width: 90px;
+    height: auto;
+}
+
+.gravestone {
+    position: absolute;
+    top: 193px;
+    left: 28px;
+    transform: rotate(-7deg);
+}
+
+.gravestone img {
+    width: 90px;
+    height: auto;
+}
+
+.tombstone {
+    position: absolute;
+    bottom: 18px;
+    left: 115px;
+    transform: rotate(10deg);
+}
+
+.tombstone img {
+    width: 103px;
+    height: auto;
+}
+
+#splashMain {
+    background-color: #874000;
+    margin: 100px auto;
+    width: 800px;
+    height: 600px;
+    text-align: center;
+}
+
+.splashHeading h1 {
+    padding-top: 20px;
+    font-size: 4rem;
+    font-family: "Creepster";
+    letter-spacing: 3px;
+}
+
+.splashOuterbox {
+    width: 100%;
+    height: 80%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.splashInnerbox {
+    background-color: #BC5F04;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 20px;
+    width: 40%;
+    height: 100%;
+}
+
+.gameLogo {
+    width: 150px;
+    height: auto;
+}
+
+.startButton {
+    width: 150px;
+    padding: 20px;
+    font-size: 2rem;
+    cursor: pointer;
+    font-family: "Creepster";
+}
+
+.instructionsTitle {
+    font-family: "Creepster";
+    font-weight: normal;
+    letter-spacing: 3px;
+    font-size: 3rem;
+}
+
+.instructionDetails {
+    font-size: 1.5rem;
+    font-family: "Fredericka the Great";
+}
+
+.modal {
+    position: fixed;
+    display: none;
+    width: 100%;
+    height: 100%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    overflow: auto;
+    background-color: rgb(0, 0, 0); /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+    z-index: 30;
+}
+
+.popUpContent {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #BC5F04;
+    margin: 0 auto;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 300px;
+    margin-top: 200px;
+}
+
+.popUpContent p {
+    text-align: center;
+    font-size: 1.5rem;
+    font-family: "Fredericka the Great";
+    padding: 20px;
+}
+
+.popUpContent h1 {
+    text-align: center;
+    font-size: 3rem;
+    font-family: "Creepster";
+    margin-bottom: 0px;
+}
+
+.endButton {
+    width: 100px;
+    padding: 10px;
+    font-size: 2rem;
+    cursor: pointer;
+    font-family: "Creepster";
+}
+
```

Comment:
#timer p margin: 0; why?

Reply:
as p has a default margin which created space between the game box and title

Reply:
chill one thats safe